superset/sql_lab.py

```diff
@@ -233,7 +233,7 @@ def execute_sql_statement(  # pylint: disable=too-many-arguments,too-many-statem
         )
     if apply_ctas:
         if not query.tmp_table_name:
-            start_dttm = datetime.fromtimestamp(query.start_time)
+            start_dttm = datetime.fromtimestamp(query.start_time / 1000)
```

Comment:
Why did this start happening? Is this happening with just a specific database?

Reply:
Just a curious nit should we enum or global the 1000 if it's used throughout?